### PR TITLE
navbar, static

### PR DESCRIPTION
**Pull request by ZefRi19**

### CODE REVIEW

File: pjbook/settings.py [hunk 1]
```diff
@@ -54,7 +54,8 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, 'templates')]
+        ,
```
Comment (lifearoundfreaks) [inline pjbook/settings.py]: Suddenly a wild coma appears.

File: pjbook/urls.py [hunk 2]
```diff
@@ -14,8 +14,17 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
```
Comment (lifearoundfreaks) [inline pjbook/urls.py]: Do not import modules unless you use them. This is often considered a bad practice.

File: pjbook/urls.py [hunk 3]
```diff
@@ -14,8 +14,17 @@
...
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', index),
 ]
+
+if settings.DEBUG:
+    urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (lifearoundfreaks) [inline pjbook/urls.py]: Do we really need that?

File: pjbook/view.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def index(request):
+    return render(request, 'base/base.html')
```
Comment (lifearoundfreaks) [inline pjbook/view.py]: I suggest learning and using class-based views right from the start.

Also, do not use base.html as a template for views, use other templates, which will extend base.html with needed content.

**Review by lifearoundfreaks — CHANGES_REQUESTED** [review]

File: pjbook/urls.py (hunk 2 above)
Comment (ZefRi19) [reply pjbook/urls.py]: done

File: pjbook/settings.py (hunk 1 above)
Comment (ZefRi19) [reply pjbook/settings.py]: done

File: pjbook/urls.py (hunk 3 above)
Comment (ZefRi19) [reply pjbook/urls.py]: done

File: pjbook/view.py (hunk 4 above)
Comment (ZefRi19) [reply pjbook/view.py]: done

**Pull request merged**